api/endpoints/community.py

```diff
@@ -37,9 +36,7 @@ def make_blueprint(con):
         "rent_burdened_2019": lambda: metric_rbu.rent_burdened(year=2019, segment="all"),
         "rent_max": lambda: metric_rbu.rent_max_burdened(),
         "rent_min": lambda: metric_rbu.rent_min_burdened(),
-        "rent_average": lambda: metric_rbu.rent_average_burden_area(),
-        "avg_speed_per_dropoff": lambda: metric_tt.get_avg_speed_per_dropoff(),
-        "avg_speed_per_pickup": lambda: metric_tt.get_avg_speed_per_pickup()
```

Comment:
Add these two average speed metrics back after the rent burdened, this is accidentally deleting them from the endpoint.